tools/deployment/package/docker-compose.base.yaml

```diff
@@ -241,31 +257,26 @@ services:
   compression-worker:
     <<: *service_defaults
     hostname: "compression_worker"
+    stop_grace_period: "60s"
     environment:
-      AWS_ACCESS_KEY_ID: "${CLP_AWS_ACCESS_KEY_ID}"
-      AWS_SECRET_ACCESS_KEY: "${CLP_AWS_SECRET_ACCESS_KEY}"
-      BROKER_URL: "amqp://${CLP_QUEUE_USER}:${CLP_QUEUE_PASS}@queue:5672"
+      BROKER_URL: "amqp://${CLP_QUEUE_USER:?error}:${CLP_QUEUE_PASS:?error}@queue:5672"
       CLP_CONFIG_PATH: "/etc/clp-config.yml"
       CLP_HOME: "/opt/clp"
       CLP_LOGGING_LEVEL: "${CLP_COMPRESSION_WORKER_LOGGING_LEVEL:-INFO}"
       CLP_LOGS_DIR: "/var/log/compression_worker"
       CLP_WORKER_LOG_PATH: "/var/log/compression_worker/worker.log"
       PYTHONPATH: "/opt/clp/lib/python3/site-packages"
       RESULT_BACKEND: >-
-        redis://default:${CLP_REDIS_PASS}@redis:6379/${CLP_REDIS_COMPRESSION_BACKEND_DB:-1}
+        redis://default:${CLP_REDIS_PASS:?error}@redis:6379/${CLP_REDIS_BACKEND_DB_COMPRESSION:-1}
     volumes:
       - *volume_aws_config_readonly
       - *volume_clp_config_readonly
-      - *volume_root_logs_readonly
-      - type: "bind"
-        source: "${CLP_ARCHIVE_OUTPUT_DIR_HOST:-./var/data/archives}"
-        target: "/var/data/archives"
-      - type: "bind"
-        source: "${CLP_ARCHIVE_OUTPUT_DIR_HOST:-./var/data/staged-archives}"
-        target: "/var/data/staged-archives"
-      - type: "bind"
-        source: "${CLP_COMPRESSION_WORKER_LOGS_DIR_HOST:-./var/log/compression_worker}"
-        target: "/var/log/compression_worker"
+      - *volume_clp_logs
+      - *volume_logs_input_readonly
+      - "${CLP_ARCHIVE_OUTPUT_DIR_HOST:-empty}:\
+${CLP_ARCHIVE_OUTPUT_DIR_HOST:-/tmp}/var/data/archives"
+      - "${CLP_STAGED_ARCHIVE_OUTPUT_DIR_HOST:-empty}:\
+${CLP_STAGED_ARCHIVE_OUTPUT_DIR_HOST:-/tmp}/var/data/staged-archives"
```

Comment:
I'm a bit confused by this. If `CLP_ARCHIVE_OUTPUT_DIR_HOST` is set, aren't we setting the mount to, for example, `var/data/archives:var/data/archives/var/data/archives`?